packages/babel-parser/test/fixtures/typescript/types/import/input.js

```diff
@@ -0,0 +1,2 @@
+let x: typeof import('./x');
```

Comment:
Q: Is `typeof import()` `TheModule` or `Promise<TheModule>`? In the second case I don't think that we should use a ts-specific node since `import()` is just a es expression.

Reply:
as a small info typeof is not mandatory for `ImportType`
```
type B = import("B").X;
```
is valid to